python/lsst/ts/observatory/control/maintel/mtcalsys.py

```diff
@@ -279,6 +345,24 @@ async def setup_laser(self, mode: LaserDetailedState) -> None:
                 f"{mode} not an acceptable LaserDetailedState [CONTINOUS, BURST, TRIGGER]"
             )
 
+        await self.change_laser_optical_configuration(optical_configuration)
+        await self.change_laser_wavelength(wavelength, use_projector)
+
+    async def laser_start_propagate(self) -> None:
+        """Start the propagation of the Tunable Laser"""
+        # get laser detailed state
+
+        await self.rem.tunablelaser.cmd_startPropagateLaser.start(
```

Comment:
How does the propagation work? This might be a good situation to add a check on a evt to check that the propagation has actually started. You can check against the detailedstate evt that the laser is propagating with the right mode before this method completes. 

Maybe using something like this https://github.com/lsst-ts/ts_observatory_control/blob/cab4bdbae2dbf41863a87eadc272396a523dc9e5/python/lsst/ts/observatory/control/auxtel/atcalsys.py#L675